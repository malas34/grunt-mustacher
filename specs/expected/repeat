<span class="odd first">Hello - 0</span>
<span class="even">Hello - 1</span>
<span class="odd">Hello - 2</span>
<span class="even last">Hello - 3</span>
