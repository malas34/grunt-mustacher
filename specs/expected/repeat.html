<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
        <title></title>
        <meta name="description" content="">
        <meta name="viewport" content="width=device-width">
    </head>
    <body>
        <h1>Repeat Helper</h1>
        <h2>Repeat 4</h2>
        <p>
            <b>Hello - 0</b>
            <b>Hello - 1</b>
            <b>Hello - 2</b>
            <b>Hello - 3</b>
        </p>
        <h2>Is First</h2>
        <p>
            <b>Hello - first 0</b>
        </p>
        <h2>Is Last</h2>
        <p>
            <b>Hello - last 3</b>
        </p>
    </body>
</html>
